parkhangs-webapp/src/features/modal/ModalMapDetail.js

```diff
@@ -75,45 +79,76 @@ class ModalMapDetail extends React.Component {
                             </div>
                         </div>
                     </div>
-                    <div className="Section">
+                    <div className="Section" style={{maxHeight: "300px", overflow: "scroll"}}>
```

Comment:
could you please pull this out into a css file? Would prefer to avoid inline css